msc_disk.c

```diff
@@ -182,18 +232,15 @@ bool tud_msc_start_stop_cb(uint8_t lun, uint8_t power_condition, bool start, boo
 
   if ( load_eject )
   {
-    if (start)
-    {
-    	ejected = false;
-      // load disk storage
-    }else
-    {
-      // unload disk storage
-      latch_ejected = true;
-      ejected = true;
-    }
+      if (start) {
+          host_ejected = false;
+          // load disk storage
+      }else {
+          host_ejected = true;
+          if (writable)

```

Comment:
Please use braces with `if`/`else`/`do`/`while` statements.   Not using braces, especially in code that uses macros (e.g., anything in C) can lead to subtle bugs.